### PR TITLE
Fix aggregator log timestamps

### PR DESCRIPTION
**Pull request by alextargov**
**Description**

Changes proposed in this pull request:
- ORD Aggregator uses JSON formatting for the logger. This causes the GCP logs to lack information about log's milliseconds. With this, all logs in a certain second have "000" milliseconds leading to messed up logs.
- This is how logs on dev-main look after this change was applied there.
image

**Related issue(s)**

- ...

**Pull Request status**

- [x] Implementation
- [ ] Unit tests
- [ ] Integration tests
- [x] `chart/compass/values.yaml` is updated 
- [ ] Mocks are regenerated, using the automated script

### CODE REVIEW

File: components/director/pkg/log/configure.go [hunk 1]
```diff
@@ -48,6 +49,15 @@ var (
 		textFormatterKey: &logrus.TextFormatter{},
 	}
 
+	supportedFormattersWithTimestamp = map[string]logrus.Formatter{
```
Comment (dzahariev) [inline components/director/pkg/log/configure.go:52]: I think is better to have only formatters with timestamps instead.

File: chart/compass/charts/ord-aggregator/templates/deployment.yaml [hunk 2]
```diff
@@ -108,6 +108,8 @@ spec:
               value: "{{ .Values.global.ordAggregator.http.retry.attempts }}"
             - name: APP_HTTP_RETRY_DELAY
               value: "{{ .Values.global.ordAggregator.http.retry.delay }}"
+            - name: APP_LOG_TIMESTAMP_FORMATTED
```
Comment (dzahariev) [inline chart/compass/charts/ord-aggregator/templates/deployment.yaml:111]: I think this is not needed. Having timestamps should be default behaviour.

**Review by dzahariev — CHANGES_REQUESTED** [review]
Please simplify this.

**Review by dzahariev — APPROVED** [review]

**Review by la4ezar — APPROVED** [review]

**Pull request merged**